modules/security-jwt/src/main/java/com/devonfw/module/security/jwt/authentication/DefaultAuthenticationTokenDetailsImpl.java

```diff
@@ -0,0 +1,93 @@
+package com.devonfw.module.security.jwt.authentication;
+
+import java.util.List;
+
+import javax.inject.Named;
+
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+
+/**
+ * Default Implementation of {@link AuthenticationTokenDetails}
+ *
+ * @since 3.3.0
+ */
+@Named
+public class DefaultAuthenticationTokenDetailsImpl implements AuthenticationTokenDetails {
```

Comment:
This class is used as a singleton spring bean but is stateful and therefore not thread-safe. This can not work.